share/github-backup-utils/ghe-restore-repositories

```diff
@@ -140,20 +143,37 @@ fi
 # rsync all the repository networks to the git server where they belong.
 # One rsync invocation per server available.
 bm_start "$(basename $0) - Restoring repository networks"
+rsync_commands=()
 for file_list in $tempdir/*.rsync; do
   if $CLUSTER; then
     server=$(basename $file_list .rsync)
   else
     server=$host
   fi
-  ghe_verbose "* Transferring repository networks to $server ..."
-  ghe-rsync -avrR --delete \
-    -e "ssh -q $opts -p $port $ssh_config_file_opt -l $user" \
-    --rsync-path="sudo -u git rsync" \
-    --files-from=$file_list \
-    "$GHE_DATA_DIR/$GHE_RESTORE_SNAPSHOT/repositories/./" \
-    "$server:$GHE_REMOTE_DATA_USER_DIR/repositories/" 1>&3
+
+  rsync_commands+=("
+    if [ -n \"$GHE_VERBOSE\" ]; then
+      echo \"* Transferring repository networks to $server ($file_list) ...\" 1>&3
+      cat $file_list
```

Comment:
Why are we catting this? You can use ghe_verbose to send something to the verbose log.

Reply:
👍 This shouldn't be here at all. As I explained previously, `ghe_verbose` isn't available in the `parallel` environment